polars/benches/rechunk.rs

```diff
@@ -0,0 +1,106 @@
+use criterion::{criterion_group, criterion_main, Criterion};
```

Comment:
I think the benchmarks (or at least the results) are interested to see in the PR posts.

We just removed the benches directory in #6273 as they were outdated and never used.

Sorry for the confusion on this one. :sweat_smile: